README.md

```diff
@@ -53,10 +53,8 @@ Currently, this project supports the following environments.
 | Ubuntu  | 1.21       | amd64        |
 | Ubuntu  | 1.22       | 386          |
 | Ubuntu  | 1.21       | 386          |
-| Linux   | 1.22       | arm64        |
-| Linux   | 1.21       | arm64        |
```

Comment:
Should we keep the arm64 Linux? I wonder why we are removing this from the support list.